deployment/base/scripts/init_data/metadata_profile_schemas/QandA_default_schema.xml

```diff
@@ -0,0 +1,76 @@
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+	<xsd:element name="metadata">
```

Comment:
AS to Guy currently it is not needed.
If it will be needed in the future we will update the schema.